### PR TITLE
🐛 Fix advanced server request example

### LINKED ISSUE #5232: The "advanced server request" example in playground is broken
### Current behavior
The "advanced server request" example in playground for the email format is broken. The amp-list XHR to https://amp.dev/documentation/examples/api/echo?__amp_source_origin=https%3A%2F%2Fplayground.amp.dev fails with HTTP 400:

```
Requests must set content-type=application/json
```

Please also provide:
- Page link: https://playground.amp.dev/?runtime=amp4email&url=https%3A%2F%2Fpreview.amp.dev%2Fdocumentation%2Fexamples%2Finteractivity-dynamic-content%2Fadvanced_server_request

/to @sebastianbenz

### PR DESCRIPTION
**Pull request by lluerich**
Fixes #5232.

### CODE REVIEW

File: examples/static/inline-examples/data/amp-list-animals.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "items": [
+    {
+      "animal": "dog"
```
Comment (sebastianbenz) [inline examples/static/inline-examples/data/amp-list-animals.json:4]: Should we list all animals?

**Review by sebastianbenz — APPROVED** [review]
Thanks for fixing!

File: examples/static/inline-examples/data/amp-list-animals.json (hunk 1 above)
Comment (lluerich) [reply examples/static/inline-examples/data/amp-list-animals.json:4]: Sounds good to me. Like this?

**Comment by sebastianbenz** [comment]
Perfect - thanks!

>

**Pull request merged**